test/integration/examples-bad/map_rect/stanc.expected

```diff
@@ -15,7 +15,7 @@ Instead supplied arguments of incompatible type: (vector, vector, real[], int[])
 
   $ ../../../../../install/default/bin/stanc bad_data_int_const.stan
 
-Semantic error at file bad_data_int_const.stan, line 21, columns 8-104:
+Semantic error at file bad_data_int_const.stan, lines 21-22, column 8:
```

Comment:
"error at file" should be "error in file".  Sorry for the randomness of prepositions.

Reply:
Done.